k8s-fluentbit/main.tf

```
@@ -0,0 +1,31 @@
+locals {
+  # this should be the default calculated name anyway, but we want to enforce it just to be sure
+  service_account_name = trimprefix(var.release == trimsuffix(var.release, "-fluentbit") ? "${var.release}-fluentbit": var.release, "-")
+
+  # extract data about kubernetes cluster
+  cloud            = var.kubernetes_cluster.specs.kubernetes.cloud
+  k8s_distribution = var.kubernetes_cluster.specs.kubernetes.distribution
+
+  helm_values = {
+    serviceAccount = {
+    name = local.service_account_name
+    }
+    annotations = var.md_metadata.default_tags
```

This is cool, I haven't seen default tags added as annotations to our other helm releases. Maybe something we want to do

---

```suggestion
    labels = var.md_metadata.default_tags
```

We actually want to use labels instead of annotations here. Labels are used for identification/querying.  Annotations are explicitly non-identifying: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/  and generally contain information regarding the resource itself, and can even have side-effects such as determining which type of load balancer gets allocated (ELB, NLB, ALB), or telling external-dns/cert-manager how to manage DNS records/certificates.